pym/_emerge/BinpkgExtractorAsync.py

```diff
@@ -39,6 +46,21 @@ def _start(self):
 			self._async_wait()
 			return
 
+		decompression_binary = varexpand(shlex_split(decomp_cmd)[0], mydict=self.env)
```

Comment:
This could be used as-is, but you would need to check for empty decompression_binary afterwards.